wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisSpeeds.java

```diff
@@ -362,6 +362,28 @@ public ChassisSpeeds div(double scalar) {
         vxMetersPerSecond / scalar, vyMetersPerSecond / scalar, omegaRadiansPerSecond / scalar);
   }
 
+  /**
+   * Checks if the ChassisSpeeds is equivalent to the other provided ChassisSpeeds.
+   *
+   * @param o The object compare this to.
+   */
+  @Override
```

Comment:
Does this need the `@Override`?

Reply:
It's best practice- `@Override` tells the compiler to enforce that the method does override an inherited method, which can catch bugs like misspellings, incorrect parameter types, etc.

However, you don't need the doc comment (since javadoc will use the doc comment for the overridden method) and should remove it, since the original doc comment is more accurate.

Reply:
It appears to be failing because of the `@Override`, any idea how to fix that?

Reply:
It's not failing because of the `@Override` annotation, it's failing because there's no `hashCode` override to go with the `equals` override.

Reply:
Ok, what should I add for the hashCode